packages/web/src/scss/components/Alert/_theme.scss

```diff
@@ -1,31 +1,23 @@
 @use 'sass:map';
-@use '@tokens' as tokens;
+@use '@global' as global-tokens;
 @use '../../settings/dictionaries';
 
-$typography: tokens.$body-medium-text-regular;
+$typography: global-tokens.$body-medium-regular;
 
-$gap: tokens.$space-600;
-$padding: tokens.$space-600;
-$border-radius: tokens.$radius-200;
-$border-style: tokens.$border-style-100;
-$border-width: tokens.$border-width-100;
+$gap: global-tokens.$space-700;
+$padding: global-tokens.$space-700;
+$border-radius: global-tokens.$radius-300;
+$border-style: solid;
```

Comment:
Shall we still keep it in the theme?

Reply:
Currently, I do this everywhere. TBH I don't care where we set it. I will keep it here as I do it elsewhere now, but we can move it in the future without problems.